Makefile

```diff
@@ -13,6 +13,20 @@ JAVA_GEN_VERSION := v0.5.4
 TESTPARALLELISM := 10
 WORKING_DIR := $(shell pwd)
 
+# Download local copy of schema-tools based on the version in .schema-tools.version
```

Comment:
Is this change consistent with what Ecosystem is doing?

Reply:
It's what was added to `pulumi-aws`. It's only run locally so that we can verify the schema changes before we push them up here.